src/components/cards/employees-card/EmployeeCard.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="employee-card">
+
+      <h4 class="employeeCardName">{{employeeFirstName}}</h4>
```

Comment:
- The css convetion should follow the DDD pattern convention, try to not repeat information
If you are inside `employee-card` is not necessary to put it again in name
eg: `first-name` instead `employeeCardName`
- The CSS class should be in `kebab-case`